ui/pages/index.js

```diff
@@ -4,10 +4,6 @@ import { Provider } from 'react-redux';
 import { HashRouter } from 'react-router-dom';
 import * as Sentry from '@sentry/browser';
 import { I18nProvider, LegacyI18nProvider } from '../contexts/i18n';
-import {
-  MetaMetricsProvider,
-  LegacyMetaMetricsProvider,
-} from '../contexts/metametrics';
 import {
   MetaMetricsProvider as NewMetaMetricsProvider,
```

Comment:
Justnoting that we should rename this to remove the 'new' syntax in your next PR along with updating to remove the .new file suffix from the other file :)

Reply:
I agree on that! :)